bindings/go/go.mod

```diff
@@ -8,4 +8,5 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
+	github.com/goproxy/goproxy v0.0.1
```

Comment:
Thanks for the PR. However, it appears you might not be fully versed in this issue. Would you prefer to tackle a different one?

Reply:
ok, i will close this pr